docs/integrations/web-servers/opentelemetry/varnish-opentelemetry.md

```diff
@@ -0,0 +1,172 @@
+
+To create a new Collector:
+
+1. Select the **Add a new Collector** option.
+2. Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.  
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Varnish-OpenTelemetry/Varnish-Collector.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) yaml file which has all the details with respect to the data which needs to be collected. For example, it specifies the location of a log file that is read and sent to the Sumo Logic platform.
+
+In this step we will be configuring the yaml required for Varnish Collection.
+
+Path of the log file configured to capture Varnish logs needs to be given here.
+
+The files are located in `/var/log/varnish/varnishncsa.log` by default. For more details please refer "Prerequisite" section of this page.
+You can add any custom fields which you want to tag along with the data ingested in sumo.
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Varnish-OpenTelemetry/Varnish-YAML.png')} alt="YAML" />
+
+### Step 3: Sending logs to Sumo
+
+Once you have the yaml file downloaded in step 2, please follow the below steps based on your environment:
+
+<Tabs
+  className="unique-tabs"
+  defaultValue="Linux"
+  values={[
+    {label: 'Linux', value: 'Linux'},
+    {label: 'macOS', value: 'macOS'},
+  ]}>
+
+<TabItem value="Linux">
+
+1.  Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Varnish instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the below command 
+```sh
+sudo systemctl restart otelcol-sumo
+```
+
+</TabItem>
+<TabItem value="macOS">
+
+1.  Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Varnish instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the below command 
+```sh
+otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --conf "glob:/etc/otelcol-sumo/conf.d/*.yaml"
+```
+
+</TabItem>
+</Tabs>
+
+After successful execution of the above command, Sumo will start receiving the data from your host machine. 
+
+This will install the app to your Sumo Logic Org. The app consists of Dashboards.
+
+Panels will start to fill automatically. It's important to note that each panel fills with data matching the time range query and received since the panel was created. Results won't immediately be available. Your data should appear in the dashboard within 20 minutes and you'll see full graphs and map.
```

Comment:
```suggestion
Panels will start to fill automatically. It's important to note that each panel fills with data matching the time range query and received since the panel was created. Results won't immediately be available. Your data should appear in the dashboard within 20 minutes and you'll see full graphs and maps.
```